src/UKHO.S100PermitService.Common/Services/PermitService.cs

```diff
@@ -98,5 +107,15 @@ private static List<Products> GetProductsList()
             };
             return productsList;
         }
+
+        [ExcludeFromCodeCoverage]
```

Comment:
This method should definitely have unit tests, and should be called something like CreateProductKeyServiceRequests(...)